ajax/libs/readremaining.js/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "readremaining.js",
+  "version": "1.0.0",
+  "filename": "output.min.js",
+  "description": "A jQuery plugin that allows you to show the user, how long will take to read the article taking in consideration the speed in which he is scrolling.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aerolab/readremaining.js"
+  },
+  "keywords": [
+    "jquery",
+    "ui",
+    "read",
+    "timing"
+  ],
+  "author": "Aerolab",
+  "license": "MIT",
+  "homepage": "https://aerolab.github.io/readremaining.js/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Aerolab/readremaining.js.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "**/+(output|readremaining)*"
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "css/**/*"
```

Comment:
I think it would be better to use `css/rr*` because we don't know what else will it be under css directory, maybe something we don't need. `rr` seems to mean "r"ead"r"emaining which is essential for this library.